sequence_processing_pipeline/templates/nuqc_job.sh

```diff
@@ -56,11 +51,7 @@ fi
 # DO NOT do this casually. Only do a clean up like this if
 # you know for sure TMPDIR is what you want.
 
-# we might got back to this TMPDIR once the slurm scheduler controls it
-#    TMPDIR=/dev/shm
-# right now, let's use ${OUTPUT}, note that each worker will create a new tmp
-# folder two lines below
-TMPDIR=${OUTPUT}
+TMPDIR=/dev/shm
```

Comment:
just to confirm /dev/shm is okay to use again?

Reply:
yes